static/app/views/issueList/overview.tsx

```diff
@@ -1215,6 +1163,32 @@ class IssueListOverview extends Component<Props, State> {
     };
   };
 
+  onStatsRequestPromise = (query: GroupStatsQuery) => {
```

Comment:
Looks like this function can be renamed now that it's not using the promise?

Reply:
Good catch, I renamed it